### DISCUSSION TITLE
Addition for the function: [ Reverse Route ]

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#304] [Q&A] [answered]
I have a "Wish". . .  Reason: an addition Button behind: [ Routes ] . . . a Button for the function: [ Reverse Route ]
Justification, a typical case. . . we are in Slovenia, also in Tuscany for the first time - no local knowledge and are looking for a tourist location for dinner on the web.
From the bungalow there, in daylight and with a good overview. Input in the navigation: from "Loacation" A to B . . . The navigation does the rest 👍. At midnight - pitch black, the return journey. . . If the Bungalow/Hotel is not listed in the favorites? the panic starts back when planning . . . 😨 "What was the name of the Village again . . .?", "which street was that again??" . . . 😳 So fiddle around with Booking.com, re-enter everything and the home it's back again. It would all be simple and easier with a Button: [ Reverse Route, now "B" to "A" ]

**Comment by WolfgangZ42** [comment, marked as answer]
That's already implemented: just tap anywhere on the route, then on "reverse route" and start navigation. 

<img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/68203ed4-4536-427a-862d-18b316705578"/>  <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/335df59a-571d-4e13-bc9a-500827605b4c"/>  <img width="200" src="https://github.com/devemux86/cruiser/assets/121806238/7defb688-4db4-41d3-85f6-94543b04d41d"/>

I think, an extra button isn't necessary.

**Reply by SF4User** [reply]
aaahh! it's that easy . . . if you understand cruiser :) I'm learning contineuos. Thank you 👍

**Comment by devemux86** [comment]
Long press on anything on the map or in the lists of various items and a menu with additional actions will open.

---

It's nice to see different app (accent) colors in your screenshots!
People are different, they like different colors, it's nice that Cruiser can satisfy you even in that.

(most operating systems offer different accent colors)
